src/Core/tests/DeviceTests/Stubs/EditorStub.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace Microsoft.Maui.DeviceTests.Stubs
+{
+	public partial class EditorStub : StubBase, IEditor
+	{
+		public string Text { get; set; }
+
+		public Color TextColor { get; set; }
```

Comment:
TextColor is here to prevent issues wrt not implementing the full interface properly